core/src/Platforms/iOS/EmbeddedWebview/AuthenticationAgentUIViewController.cs

```diff
@@ -64,90 +63,93 @@ public override void ViewDidLoad()
 
             View.BackgroundColor = UIColor.White;
 
-            webView = new UIWebView(View.Bounds);
-            webView.ShouldStartLoad = (wView, request, navType) =>
+            WKWebViewConfiguration webConfiguration = new WKWebViewConfiguration();
+
+            wkWebView = new WKWebView(View.Bounds, webConfiguration);
+            NSUrl url = wkWebView.Url;
+
+            wkWebView.LoadRequest(new NSUrlRequest(url));
+
+            if (LoadView(url) == true)
             {
-                if (request == null)
-                {
-                    return true;
-                }
+                wkWebView.StopLoading();
 
-                string requestUrlString = request.Url.ToString();
+                // If the title is too long, iOS automatically truncates it and adds ...
```

Comment:
In the previous implementation, if the title was empty we used "Sing In". Why aren't we doing this anymore?